app/dashboard/static/js/app/view-tests-board-job-kernel.2017.7.2.js

```diff
@@ -0,0 +1,534 @@
+        }
+
+        results = response.result;
+        if (results.length > 0) {
+            // Parse all the results and update a global object with
+            // the operation IDs and the count found.
+            results.forEach(_parseOperationsResult);
+            // Invalidate the cells in column #2 before updating the DOM
+            // elements. In this way we have the correct 'filter' values in the
+            // global object that we can use to provide the search parameters.
+            labTable.invalidateColumn(2);
+            // Now update the DOM with the results.
+            Object.keys(gTableCount).forEach(_updateTable);
+
+            // Bind a new function to the draw event of the table.
+            labTable.addDrawEvent(updateBatchCount);
+        }
+    }
+
+    function getBatchCount(lab, labTable, response) {
+        var batchOps;
+        var deferred;
+        var suiteId;
+        var queryStr;
+        var results;
+
+        function _createOp(result) {
+        	suiteId = result._id.$oid;
+            // TODO When sorting check if more than 1 result print conflict error
+            batchOps.push({
+                method: 'GET',
+                operation_id: lab + '-sets-count-' + suiteId,
+                resource: 'count',
+                document: 'test_set',
+                query: queryStr + suiteId
+            });
+            batchOps.push({
+                method: 'GET',
+                operation_id: lab + '-cases-total-count-' + suiteId,
+                resource: 'count',
+                document: 'test_case',
+                query: queryStr + suiteId
+            });
+
+            batchOps.push({
+                method: 'GET',
+                operation_id: lab + '-cases-success-count-' + suiteId,
+                resource: 'count',
+                document: 'test_case',
+                query: queryStr + suiteId + '&status=PASS'
+            });
+
+            batchOps.push({
+                method: 'GET',
+                operation_id: lab + '-cases-fail-count-' + suiteId,
+                resource: 'count',
+                document: 'test_case',
+                query: queryStr + suiteId + '&status=FAIL'
+            });
+
+            batchOps.push({
+                method: 'GET',
+                operation_id: lab + '-cases-unknown-count-' + suiteId,
+                resource: 'count',
+                document: 'test_case',
+                query: queryStr + suiteId + '&status=OFFLINE&status=UNKNOWN'
+            });
+        }
+
+        results = response.result;
+        if (results.length > 0) {
+            batchOps = [];
+            queryStr = 'test_suite_id=';
+            results.forEach(_createOp);
+
+            deferred = request.post(
+                '/_ajax/batch', JSON.stringify({batch: batchOps}));
+
+            $.when(deferred)
+                .fail(error.error, getBatchCountFail)
+                .done(function(response) {
```

Comment:
You need to change this variable name, since `response` is already defined in the outer scope.
Even `resultz` would be fine, and pass it to the `getBatchCountDone` function call.